From f774f9a7dcb44ffaa5764a22ff487baf03063025 Mon Sep 17 00:00:00 2001
From: Philippe Gerum <rpm@xenomai.org>
Date: Thu, 2 Oct 2025 12:33:38 +0200
Subject: [PATCH 0983/1013] evl/net: net: clarify names of sysfs attributes

Routes and ARP (table) attributes are all IPv4-related, write-only,
for the purpose of flushing the respective tables. Make this obvious
by naming.

Signed-off-by: Philippe Gerum <rpm@xenomai.org>
---
 kernel/evl/net/net.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/kernel/evl/net/net.c b/kernel/evl/net/net.c
index 9a0728f01cf6..aa7eefc4f9c1 100644
--- a/kernel/evl/net/net.c
+++ b/kernel/evl/net/net.c
@@ -183,7 +183,7 @@ static ssize_t vlans_store(struct device *dev,
 }
 static DEVICE_ATTR_RW(vlans);
 
-static ssize_t ipv4_routes_store(struct device *dev,
+static ssize_t ipv4_flush_routes_store(struct device *dev,
 				struct device_attribute *attr,
 				const char *buf, size_t count)
 {
@@ -193,9 +193,9 @@ static ssize_t ipv4_routes_store(struct device *dev,
 
 	return count;
 }
-static DEVICE_ATTR_WO(ipv4_routes);
+static DEVICE_ATTR_WO(ipv4_flush_routes);
 
-static ssize_t arp_store(struct device *dev,
+static ssize_t ipv4_flush_arp_store(struct device *dev,
 			struct device_attribute *attr,
 			const char *buf, size_t count)
 {
@@ -205,12 +205,12 @@ static ssize_t arp_store(struct device *dev,
 
 	return count;
 }
-static DEVICE_ATTR_WO(arp);
+static DEVICE_ATTR_WO(ipv4_flush_arp);
 
 static struct attribute *net_attrs[] = {
 	&dev_attr_vlans.attr,
-	&dev_attr_ipv4_routes.attr,
-	&dev_attr_arp.attr,
+	&dev_attr_ipv4_flush_routes.attr,
+	&dev_attr_ipv4_flush_arp.attr,
 	NULL,
 };
 ATTRIBUTE_GROUPS(net);
-- 
2.30.2

